pandas/io/html.py

```diff
@@ -920,6 +921,7 @@ def _parse(flavor, io, match, attrs, encoding, displayed_only, **kwargs):
     return ret
 
 
+@deprecate_nonkeyword_arguments(version="1.1")
```

Comment:
Usually we go at least 3 releases, so this would be 1.4

Reply:
Thank you, changed it.